src/graph_compute/phase_scheduling.h

```diff
@@ -113,7 +114,11 @@ static inline void calculateNodeDependenciesChunk(vertex_t * const nodes,
   printf("InterChunkDependencies: %lu\n",
     static_cast<uint64_t>(cntDependencies));
   })
-  scheddata->dependentEdges = new (std::nothrow) vid_t[cntDependencies+1]();
+  //  9 bits -> 4KB page granularity
+  numaInit_t numaInit(NUM_CORES,
+                      std::min(9, CHUNK_BITS), static_cast<bool>(NUMA_INIT));
```

Comment:
The `min` part feels like it should be built into the init method itself, rather than existing at each call site.